src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -350,6 +351,13 @@ export type OncokbCancerGene = {
     isCancerGene: boolean;
 };
 
+export enum BinMethodOption {
+    QUARTILE = 'QUARTILE',
+    MEDIAN = 'MEDIAN',
+    GENERATE = 'GENERATE',
+    CUSTOM = 'CUSTOM',
+}
+
```

Comment:
This data is represented in GenomicDataBinFilter.binMethod. Better remove it and update all references.

Reply:
Not working because GenomicDataBinFilter is a type